websites/R/Repeat/presence.ts

```diff
@@ -3,7 +3,7 @@ let sartist, strack, slisteners, sdj;
 
 function newStats(): Promise<void> {
   window
-    .fetch("https://stats.repeat.pw")
+    .fetch("https://repeat.bozzy.online/api/nowplaying/1")
```

Comment:
You may only make HTTP/HTTPS requests to `premid.app` or the presence website API. If you are using external domains, you will be required to explain why it is necessary.

bozzy.online is not the presence website, so you need an api from repeat.pw or repeatradio.net